frameworks/Erlang/chicagoboss/benchmark_config.json

```diff
@@ -0,0 +1,23 @@
+{
+  "framework": "chicagoboss",
+  "tests": [{
+    "default": {
+      "setup_file": "setup",
+      "plaintext_url": "/plaintext",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "MySQL",
+      "framework": "chicagoboss",
+      "language": "Erlang",
+      "orm": "Raw",
+      "platform": "cowboy",
```

Comment:
Cannot help but notice that the classification is `Platform`, but under platform, it says `cowboy`; which is it?